2022-08-12 13:15:17,349 INFO    MainThread:164602 [wandb_setup.py:_flush():76] Configure stats pid to 164602
2022-08-12 13:15:17,349 INFO    MainThread:164602 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2022-08-12 13:15:17,349 INFO    MainThread:164602 [wandb_setup.py:_flush():76] Loading settings from /home/compu/jh/project/colon_compare/scripts/logs/evaluations/runs/colon/voting/2022-08-12_13-14-33/wandb/settings
2022-08-12 13:15:17,349 INFO    MainThread:164602 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2022-08-12 13:15:17,349 WARNING MainThread:164602 [wandb_setup.py:_flush():76] Could not find program at ../test.py
2022-08-12 13:15:17,349 INFO    MainThread:164602 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../test.py'}
2022-08-12 13:15:17,349 INFO    MainThread:164602 [wandb_init.py:_log_setup():461] Logging user logs to ./wandb/run-20220812_131517-26ryzrn2/logs/debug.log
2022-08-12 13:15:17,350 INFO    MainThread:164602 [wandb_init.py:_log_setup():462] Logging internal logs to ./wandb/run-20220812_131517-26ryzrn2/logs/debug-internal.log
2022-08-12 13:15:17,350 INFO    MainThread:164602 [wandb_init.py:init():495] calling init triggers
2022-08-12 13:15:17,350 INFO    MainThread:164602 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2022-08-12 13:15:17,350 INFO    MainThread:164602 [wandb_init.py:init():548] starting backend
2022-08-12 13:15:17,350 INFO    MainThread:164602 [wandb_init.py:init():552] setting up manager
2022-08-12 13:15:17,352 INFO    MainThread:164602 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-12 13:15:17,353 INFO    MainThread:164602 [wandb_init.py:init():558] backend started and connected
2022-08-12 13:15:17,360 INFO    MainThread:164602 [wandb_init.py:init():636] updated telemetry
2022-08-12 13:15:17,537 INFO    MainThread:164602 [wandb_init.py:init():667] communicating run to backend with 30 second timeout
2022-08-12 13:15:18,322 INFO    MainThread:164602 [wandb_run.py:_on_init():1982] communicating current version
2022-08-12 13:15:18,403 INFO    MainThread:164602 [wandb_run.py:_on_init():1986] got version response 
2022-08-12 13:15:18,403 INFO    MainThread:164602 [wandb_init.py:init():700] starting run threads in backend
2022-08-12 13:15:18,803 INFO    MainThread:164602 [wandb_run.py:_console_start():1962] atexit reg
2022-08-12 13:15:18,804 INFO    MainThread:164602 [wandb_run.py:_redirect():1820] redirect: SettingsConsole.WRAP_RAW
2022-08-12 13:15:18,804 INFO    MainThread:164602 [wandb_run.py:_redirect():1885] Wrapping output streams.
2022-08-12 13:15:18,804 INFO    MainThread:164602 [wandb_run.py:_redirect():1907] Redirects installed.
2022-08-12 13:15:18,805 INFO    MainThread:164602 [wandb_init.py:init():732] run started, returning control to user process
2022-08-12 13:15:24,426 INFO    MainThread:164602 [wandb_run.py:_config_callback():1152] config_cb None None {'ckpt_path': '/home/compu/jh/project/colon_compare/scripts/logs/experiments/runs/colon/classify4/2022-08-11_18-04-24/checkpoints/epoch_006.ckpt'}
2022-08-12 13:16:27,982 WARNING MsgRouterThr:164602 [router.py:message_loop():76] message_loop has been closed
